frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -266,6 +268,32 @@ It provides access to specific modules exposed by the plugin. It's loaded right
 `exposed-barUtils-chunk.js` is the generated webpack chunk for `barUtils` exposed module. It's loaded
 via the plugin entry chunk (`plugin-entry.js`) when needed.
 
+Plugins may also include other assets, such as JSON localization files that follow the general pattern
+`locales/<lang>/plugin__<plugin-name>.json` or static images referenced from the plugin code.
+
+## Serving plugin assets
+
+Dynamic plugins are deployed on a cluster via OLM operators. Each plugin deployment should include a web
+server that hosts the [generated assets](#generated-assets) of the given plugin.
+
+**Important!** Make sure to configure your plugin web server to use MIME type `text/javascript` for all
+JS assets to avoid any issues with the `X-Content-Type-Options: nosniff` security header used by Console
+when fetching plugin assets via Bridge URL `/api/plugins/<plugin-name>`.
+
+Requests for JS assets not served with the correct MIME type can be blocked by the browser, breaking your
+plugin's functionality.
+
+### Caching
+
+The following table lists important plugin assets that must NOT be cached by the plugin web server.
```

Comment:
Yes, AFAIK the Bridge plugin asset proxy endpoint does not do anything special in this regard. cc @jhadvig

Reply:
I know @TheRealJon has fixed some caching bugs on the console side. Can we confirm it's still a limitation?

I believe we should be able to handle caching as long as it's done properly by the plugin. If this is really a restriction, we should look at adding cache-busting parameters to the console request to avoid the problem.

Reply:
@spadgett Looking into this, all plugin entry scripts (listed via `loadScripts` in the plugin manifest) are loaded using `?cacheBuster` query parameter to circumvent potential caching issues on the server side.

Given that JSON files should be generally OK to cache (assuming we don't expect their content to change), I think I can remove the whole "Caching" section above then.